src/content/ja/2019/media.md

```diff
@@ -0,0 +1,413 @@
+---
+part_number: I
+chapter_number: 4
+title: メディア
+description: 2019年版Web Almanacのメディアの章では、画像ファイルのサイズとフォーマット、レスポンシブ画像、クライアントのヒント、遅延読み込み、アクセシビリティ、動画を取り上げています。
+authors: [colinbendell, dougsillars]
+reviewers: [ahmadawais, eeeps]
+translators: [ksakae]
+discuss: 1759
+results: https://docs.google.com/spreadsheets/d/1hj9bY6JJZfV9yrXHsoCRYuG8t8bR-CHuuD98zXV7BBQ/
+queries: 04_Media
+published: 2019-11-11T00:00:00.000Z
+last_updated: 2019-11-23T00:00:00.000Z
```

Comment:
Could we change this? Want to try an experiment to see if this causes it to get indexed faster:

```suggestion
last_updated: 2020-05-02T00:00:00.000Z
```

Reply:
I don't understand experiment, but I changed.

Reply:
We've been talking on the [Web Almanac slack channel](https://httparchive.slack.com/archives/CGFLV2S06) about how some of the translations are taking a while to get indexed by Google. Wondering if this will help. Will monitor and see...